branding/src/main/scala/com/normation/plugins/branding/BrandingConfService.scala

```diff
@@ -0,0 +1,124 @@
+/*
+*************************************************************************************
+* Copyright 2018 Normation SAS
+*************************************************************************************
+*
+* This file is part of Rudder.
+*
+* Rudder is free software: you can redistribute it and/or modify
+* it under the terms of the GNU General Public License as published by
+* the Free Software Foundation, either version 3 of the License, or
+* (at your option) any later version.
+*
+* In accordance with the terms of section 7 (7. Additional Terms.) of
+* the GNU General Public License version 3, the copyright holders add
+* the following Additional permissions:
+* Notwithstanding to the terms of section 5 (5. Conveying Modified Source
+* Versions) and 6 (6. Conveying Non-Source Forms.) of the GNU General
+* Public License version 3, when you create a Related Module, this
+* Related Module is not considered as a part of the work and may be
+* distributed under the license agreement of your choice.
+* A "Related Module" means a set of sources files including their
+* documentation that, without modification of the Source Code, enables
+* supplementary functions or services in addition to those offered by
+* the Software.
+*
+* Rudder is distributed in the hope that it will be useful,
+* but WITHOUT ANY WARRANTY; without even the implied warranty of
+* MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+* GNU General Public License for more details.
+*
+* You should have received a copy of the GNU General Public License
+* along with Rudder.  If not, see <http://www.gnu.org/licenses/>.
+
+*
+*************************************************************************************
+*/
+
+package com.normation.plugins.branding
+
+import java.nio.charset.StandardCharsets
+import java.nio.file.{Files, Paths}
+
+import net.liftweb.common._
+
+import scala.util.{Success, Try, Failure => CatchException}
+
+
+class BrandingConfService extends Loggable {
+
+  private[this] val initialValue =
+    BrandingConf(
+        true
+      , Color(204,0,0,1)
+      , true
+      , "Production"
+      , Color(255,255,255,1)
+      , true
+      , true
+      , true
+      , true
+      , true
+      , true
+      , true
+      , "Welcome, please sign in:"
+    )
+
+
+  val configFilePath = "/var/rudder/plugins/branding/configuration.json"
```

Comment:
I think we should have a configuration variable in RudderConfig for "pluginsPath" to avoid having to change it in 12 millions places if it changes at some point. And for normalization in all cases.